style.css

```diff
@@ -43,10 +43,63 @@ body {
   padding: 25px;
 }
 
+.rogue {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background-color: #6070ff;
+  margin-top: 0;
+  mix-blend-mode: multiply;
+  list-style: none;
+  padding-left: 0;
+  z-index: 1;
+}
+
+.rogue ul {
+  list-style: none;
+  width: 80%;
+  margin-top: 80px;
+}
+
+.blend-mode {
+  filter: blur(8px);
+}
+
+.ham-li {
+  display: flex;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  margin-bottom: 8px;
```

Comment:
- [ ] Kindly consider increasing the margin between the links here and try to make it identical to the figma design as our design should follow the Figma design in `(e.g., font, colors, images, text, margins).` and it is a project requirement.